docs/support/troubleshoot/setuptroubleshoot.mdx

```diff
@@ -1,37 +1,33 @@
-# ESP32 setup
+# Firmware flashing
 
-## Problems with setting up your ESP32 with Toit firmware
+## Problems flashing your ESP32
 
-<Expandable title="Do you have problems with the initial provisioning your ESP32?">
+<Expandable title="Do you have problems with the installing the firmware on your ESP32?">
 
 <Tabs>
 <Tab label="YES">
 
 - Some hardware models need physical interaction with the device during initial provisioning - like pushing a button.
 
-- If provisioning doesn't work, consult the documentation for your ESP32 chip.
+- If flashing doesn't work, consult the documentation for your ESP32 chip.
 
-- If you have timeout problems when provisioning, some hardware models require you to [hold down BOOT button](https://randomnerdtutorials.com/solved-failed-to-connect-to-esp32-timed-out-waiting-for-packet-header/).
+- If you have timeout problems when flashing, some hardware models require you to [hold down BOOT button](https://randomnerdtutorials.com/solved-failed-to-connect-to-esp32-timed-out-waiting-for-packet-header/).
   Read more info about the bootloader mode [here](https://docs.espressif.com/projects/esptool/en/latest/esp32/advanced-topics/boot-mode-selection.html).
 
-- If your device wasn't able to connect via WiFi to the Toit cloud during the provisioning, it is not yet claimed. In this state the device is provisioned with the Toit firmware,
-  but is not yet added to your Toit project. Follow the steps [here](../claimtroubleshoot) to manually claim the device once it has managed to go
-  online.
-
-- If the provision command fails during the first part of the provisioning process, the hardware ID (if any) is written to the local disk if possible and will be reused the next time the command is executed on the same computer.
-
-- If the provision command fails, consider trying with a lower baud rate (here half the default baud rate):
+- If the flashing fails, consider trying with a lower baud rate (here half the default baud rate):
 
 ```shell
-toit serial provision --baud 460800 -p wifi.ssid="MY_SSID" -p wifi.password="my_password"
+jag flash --baud 460800 --wifi-ssid="MY_SSID" --wifi-password="MY_PASSWORD"
 ```
 
 </Tab>
 
 <Tab label="NO">
 
-If you have provisioned your ESP32 but the claiming step failed, and you therefore cannot see the device in the list of devices in the Toit console,
-please follow the troubleshooting tips [here](../claimtroubleshoot).
+If Jaguar cannot find your device after having flashed it, you may be running into a network issue. Check that you development
+machine is on the same network as the device and that no firewall is preventing the communications.
```

Comment:
~~~suggestion
machine is on the same network as the device and that no firewall is preventing the communications.
You can help Jaguar to find your device by giving it the device's IP address
(which is printed at boot time). For example:
```
jag scan 192.168.1.42
```
~~~